City-of-Turku/kada
# PORI-221: Keywords breaking landing page slideshow

Keywords breaking landing page slideshow. Carousel was missing a one checkbox that limited the filter to taxonomy correctly.

To test:
1. drush fra -y && drush cc all
2. create landing-page with an target audience, then liftup with that same audience and flagit to carousel
3. make sure you added keywords to the landing-page
4. confirm that liftup or liftups are visible on the page
